config/config.go

```diff
@@ -324,6 +325,37 @@ func (cfg BaseConfig) ValidateBasic() error {
 	default:
 		return errors.New("unknown log_format (must be 'plain' or 'json')")
 	}
+
+	switch cfg.ProxyApp {
+	case "kvstore",
```

Comment:
These values are the ones in `DefaultClientCreator` right ? Would be possible to have a more idiomatic `enum` way to check them? Like

type ProxyAppType int

const (
    KVStore ProxyAppType = iota
    KVStoreUnsync
    E2E
    ...
)

Reply:
This can be a constant []string in any case, just check if it matches any of them.